qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -679,16 +679,17 @@ public void allowsUppercaseAbbreviationExceptions() throws Exception {
      * @throws Exception If error
      */
     @Test
+    @SuppressWarnings("unchecked")
```

Comment:
@mkordas I don't think so, take a look other tests in CheckstyleValidatorTest class